compiler/util/src/org/jetbrains/kotlin/config/LanguageVersionSettings.kt

```diff
@@ -144,6 +144,7 @@ enum class LanguageFeature(
 
     AllowSealedInheritorsInDifferentFilesOfSamePackage(KOTLIN_1_5),
     SealedInterfaces(KOTLIN_1_5),
+    PackagePrivateFileClassesWithAllPrivateMembers(KOTLIN_1_5, defaultState = State.DISABLED),
```

Comment:
Could you please add a comment mentioning that this is about KT-10884? Otherwise looks good :).